docs/index.md

```diff
@@ -58,4 +58,20 @@ Exercise #2: Flagging GNSS-A travel time residuals
 
 Exercise #3: Exploring GNSS-A metadata
 
-Exercise #4: Velocity estimation
\ No newline at end of file
+Exercise #4: Velocity estimation
+
+## Running the Exercises
+
+The quickest way to get started without manual installation is to use
+[GitHub Codespaces](https://github.com/features/codespaces) - "a development environment that's hosted in the cloud". This allows you to run the workshop exercises directly in your browser without needing to set up a local environment.
+In order to access the Codespace, you need to register an account with [GitHub](github.com).
+
+:::{important}
+When clicking the button below, if you have any current codespace,
+we recommend deleting it to ensure you have the latest
+changes from the workshop repository.
```

Comment:
```suggestion
changes from the workshop repository.

Additionally, once codespace is rendered, please wait until the `README.md` file is loaded to ensure that setup is complete.
```

Reply:
@johnbdesanto what do you think of the verbiage above? If that looks good, I'll commit the change. Thanks.